TheFourFifths/consus
# Get all models test

### Summary

Functional test for getting all models using server api

### Checklist


### How to Review

1. Pull
2. npm install, npm test
3. check that the test runs and passes
4. Verify test is testing what it should test. 

### Links

- [TFF-141](https://msoese.atlassian.net/browse/TFF-141)
